client/src/reducers/documentsReducer.js

```diff
@@ -0,0 +1,13 @@
+export default function documentsReducer(state, action) {
+  const newState = Object.assign({}, state);
+  switch (action.type) {
+    case 'FETCH_DOCUMENTS_SUCCESS':
+      newState.count = action.payload.documents.length;
+      newState.documents = action.payload.documents;
+      break;
+
+    default:
+      break;
```

Comment:
Added that to take care of an ESLint error that was complaining about the absence of a default case.